src/ontology/uberon-edit.obo

```diff
@@ -224043,7 +224043,7 @@ def: "A group of neurons in the floor of the fourth ventricle, in close proximit
 synonym: "nuclei perihypoglossales" EXACT [WikipediaVersioned:Perihypoglossal_nuclei&oldid=821664008]
 synonym: "perihypoglossal complex" EXACT [WikipediaVersioned:Perihypoglossal_nuclei&oldid=821664008]
 synonym: "perihypoglossal nuclear complex" EXACT [WikipediaVersioned:Perihypoglossal_nuclei&oldid=821664008]
-synonym: "PHY" EXACT [MBA:154]
+synonym: "PHY" RELATED ABBREVIATION [MBA:154]
```

Comment:
```suggestion
synonym: "PHY" EXACT ABBREVIATION [MBA:154]
```

Reply:
That helps a lot . Thanks.

Shouldn't the other (UBERON:2000438 "parhypural") treated the same way?